libraries/chain/include/eosio/chain/types.hpp

```diff
@@ -99,25 +99,19 @@ namespace eosio::chain {
     * serialization and to/from variant
     */
    class shared_blob : public shared_string {
-      public:
-         shared_blob() = delete;
-         shared_blob(shared_blob&&) = default;
-
-         shared_blob(const shared_blob& s) = default;
-
+   public:
+      shared_blob() = default;
 
-         shared_blob& operator=(const shared_blob& s) = default;
+      shared_blob(shared_blob&&) = default;
+      shared_blob(const shared_blob& s) = default;
 
-         shared_blob& operator=(shared_blob&& ) = default;
+      shared_blob(std::string_view s) : shared_string(s) {}
```

Comment:
`explicit`

Reply:
done. I used placement new for the construction to make sure there isn't a copy.